src/core/coap/secure_coap_server.cpp

```diff
@@ -115,6 +115,11 @@ void SecureServer::Receive(Message &aMessage, const Ip6::MessageInfo &aMessageIn
 
     if (!mNetif.GetDtls().IsStarted())
     {
+        Ip6::SockAddr sockAddr;
```

Comment:
It calls the default constructor, which initializes this object. And there's no other explicitly defined constructor.
```
/**
 * This class implements a socket address.
 *
 */
class SockAddr: public otSockAddr
{
public:
    /**
     * This constructor initializes the object.
     *
     */
    SockAddr(void) { memset(&mAddress, 0, sizeof(mAddress)), mPort = 0, mScopeId = 0; }
```